src/Sentry/Infrastructure/DefaultDiagnosticLogger.uwp.cs

```diff
@@ -0,0 +1,9 @@
+#if WINDOWS_UWP
+namespace Sentry.Infrastructure
+{
+    internal static class DefaultDiagnosticLogger
+    {
+        public static DebugDiagnosticLogger Get(SentryLevel minimalLevel) => new DebugDiagnosticLogger(minimalLevel);
```

Comment:
Given that we might change this not only based on this `#ifdef` I think we need a different approach here.

Maybe for now inline this in the options ctor?

So e don't need these 2 new files just for the sake of deciding whether to use Debug or Console diagnostic loggers